components/net/tests/fetch.rs

```diff
@@ -1356,7 +1356,7 @@ fn test_fetch_with_devtools() {
     let httpresponse = DevtoolsHttpResponse {
         headers: Some(response_headers),
         status: HttpStatus::default(),
-        body: None,
+        body: devhttpresponse.clone().body,
```

Comment:
This should be MESSAGE; right now we're comparing the body retrieved in the test against itself.